assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -302,6 +308,22 @@ export default function SetupUsingProxyWithSignIn() {
 
 												<p className="googlesitekit-setup__description">
 													{ description }
+													{ learnMore && (
+														<Fragment>
+															{ ' ' }
```

Comment:
We shouldn't use text like this to separate a description text and "Learn more" text, because this `" "` becomes unlocalisable by translators. Instead, this should now be:

```js
createInterpolateElement(
  sprintf(
    __( '%1$s <Link>%2$s</Link>' ),
    description,
    __( 'Learn more' ) ),
  {
    Link: <Link href={ learnMore } external />
  }
)
```

It's kind of a pain but it's the most accurate way to render this content. 😅

Reply:
Addressed, thank you, @tofumatt!